faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -163,6 +165,16 @@ object DateV {
   def apply(value: String): DateV = DateV(LocalDate.parse(value))
 }
 
+case class BytesV(@(JsonIgnore @param @field @getter) bytes: IndexedSeq[Byte]) extends ScalarValue {
```

Comment:
because `Array[Byte]` don't implement `equals()` and since it's a case class two instances with same values would never be equals. Also, as far as I could search, people don't advise to override equals and hashCode in case classes.